SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -748,6 +748,26 @@ FVector USpatialActorChannel::GetActorSpatialPosition(AActor* InActor)
 	return FRepMovement::RebaseOntoZeroOrigin(Location, InActor);
 }
 
+void USpatialActorChannel::RemoveRepNotifiesWithUnresolvedObjs(TArray<UProperty*>& RepNotifies, const FRepLayout& RepLayout, const FObjectReferencesMap& RefMap, UObject* Object)
+{
+	// Prevent rep notify callbacks from being issued when unresolved obj references exist inside structures.
+	// This prevents undefined behaviour when engine rep callbacks are issued where they don't expect unresolved objects in native flow.
+	RepNotifies.RemoveAll([&](UProperty* Property)
+	{
+		for (auto& ObjRef : RefMap)
+		{
+			bool bIsSameRepNotify = RepLayout.Parents[ObjRef.Value.ParentIndex].Property == Property;
+			bool bIsArray = RepLayout.Parents[ObjRef.Value.ParentIndex].Property->ArrayDim > 1;
+			if (bIsSameRepNotify && !bIsArray)
+			{
+				UE_LOG(LogSpatialActorChannel, Warning, TEXT("RepNotify %s on %s ignored due to unresolved Actor"), *Property->GetName(), *Object->GetName());
```

Comment:
Any reason this has to be a warning instead of Verbose? I'm not sure what a user can do about this.